### PR TITLE
Camera

### PR DESCRIPTION
**Pull request by linyh0207**
After the user clicks on the Camera icon from the buttonTabNavigator, it will open up the camera screen. Camera Screen has a button to take a picture and option to flip to the back or front camera. Also, a small overlay message display on the button of the screen. After the picture has been taken, it will change to add post view. It will display the user's current location (hard code) with an edit button. Once the edit button clicked, it will switch to the search bar (text placeholder). There will be discount price and regular price text input, tag entry bar placeholder, and cancel button to trigger the camera view again.

### CODE REVIEW

File: screens/AddPostScreen.js [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+                );
+              }}
+            >
+              <Text style={{ fontSize: 18, marginBottom: 10, color: 'white' }}> Flip </Text>
+            </TouchableOpacity>
+            <TouchableOpacity style={[t.selfCenter]} onPress={takePicture}>
+              <View
+                style={{
+                  borderWidth: 2,
+                  borderRadius: '50%',
```
Comment (Michael-Xie) [inline screens/AddPostScreen.js:83]: Screenshot from 2020-06-23 08-08-59

might be related to this too: https://github.com/facebook/react-native/issues/23219

File: App.js [hunk 2]
```diff
@@ -26,8 +26,13 @@ export default function App(props) {
       <View style={[t.flex1, t.bgWhite]}>
         {Platform.OS === 'ios' && <StatusBar barStyle="dark-content" />}
         <NavigationContainer>
-          <Stack.Navigator initialRouteName={isAuth ? 'Home' : 'Onboarding'}>
-            <Stack.Screen name="Onboarding" component={OnboardingScreen} />
+          <Stack.Navigator initialRouteName={isAuth ? 'Home' : 'Onboarding'} headerMode="none">
+            {/* Onboarding screen do not need the header */}
```
[the remark this answers is not in the document]
Comment (Michael-Xie) [reply App.js:30]: top menu button is unreachable as a result

File: screens/AddPostScreen.js (hunk 1 above)
Comment (linyh0207) [reply screens/AddPostScreen.js]: It's an issue related to Android. I fixed with break it down the borderRadius: https://github.com/react-native-community/react-native-video/issues/685.

File: App.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (linyh0207) [reply App.js]: Fix the error by applying SafeAreaView for each screen. https://reactnavigation.org/docs/handling-safe-area/

**Comment by Michael-Xie** [comment]
Yay, it works :D
- good that there is a cancel button
- there should be a 'Post' button too
- a minor fix for camera, the live view sees the stretched version and the taken picture returns back to normal

**Comment by linyh0207** [comment]
- Added 'Post' button which reset the Camera Screen view and then navigator back to product main screen. 
- The display image was originally displayed at 75%, it is now fixed to 100%.

**Pull request merged**